arangosh/Shell/ClientFeature.cpp

```diff
@@ -234,6 +234,25 @@ void ClientFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
     FATAL_ERROR_EXIT();
   }
 
+  if (!_endpoint.empty() &&
+      (_endpoint != "none") &&
+      (_endpoint != Endpoint::defaultEndpoint(Endpoint::TransportType::HTTP))) {
+    std::unique_ptr<Endpoint> endpoint(Endpoint::clientFactory(_endpoint));
+    if (endpoint.get() == nullptr) {
```

Comment:
apparently this breaks VST client connections, so we should change this first before we merge it.

Reply:
Fixed